bind.go

```diff
@@ -60,7 +81,11 @@ func (l *Conn) SimpleBind(simpleBindRequest *SimpleBindRequest) (*SimpleBindResu
 	}
 	defer l.finishMessage(messageID)
 
-	packet = <-channel
+	select {
+	case packet = <-channel:
+	case <-time.After(DefaultTimeout):
```

Comment:
imposing a timeout on an operation that previously was server-response-bound is sort of a breaking change, especially if it is using a package variable that is documented as only applying to Dial/DialTLS